### PR TITLE
checkable card

### PR DESCRIPTION
**Pull request by diegozubieta95**

### CODE REVIEW

File: addon/components/eui-checkable-card/index.hbs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{{#let (has-block "label") (arg-or-default @id (unique-id)) as |hasLabelBlock id|}}
+  <div
+    class="euiCheckableCard euiCheckableCard--shadow euiCheckableCard--borderRadiusMedium"
```
Comment (betocantu93) [inline addon/components/eui-checkable-card/index.hbs:3]: I think we are missing some conditional classes, take a look here: 

https://github.com/elastic/eui/blob/75db772ef27ba01a553b1600921bcddca98b03b6/src/components/card/checkable_card/checkable_card.tsx#L66

We need to make sure we comply with the spec

**Review by betocantu93 — CHANGES_REQUESTED** [review]
We need to

File: addon/components/eui-checkable-card/index.hbs (hunk 1 above)
Comment (diegozubieta95) [reply addon/components/eui-checkable-card/index.hbs]: @betocantu93  I just added some conditional classes what do you think?????

File: addon/components/eui-checkable-card/index.hbs [hunk 2]
```diff
@@ -1,6 +1,12 @@
 {{#let (has-block "label") (arg-or-default @id (unique-id)) as |hasLabelBlock id|}}
   <div
-    class="euiCheckableCard euiCheckableCard--shadow euiCheckableCard--borderRadiusMedium"
+    class={{class-names
+      "euiCheckableCard"
+      "euiCheckableCard--shadow"
```
Comment (betocantu93) [inline addon/components/eui-checkable-card/index.hbs:5]: I think this class must be euiCheckableCard--hasShadow?

**Review by betocantu93 — DISMISSED** [review]

File: addon/components/eui-checkable-card/index.hbs (hunk 2 above)
Comment (diegozubieta95) [reply addon/components/eui-checkable-card/index.hbs:5]: Ok, let me see the difference

**Pull request merged**